test/unit/specs/components/share-list.spec.js

```diff
@@ -0,0 +1,9 @@
+import ShareList from '@/components/ShareList';
+import render from '../../test-utils/render';
+
+describe('ShareList', () => {
+  it('should render correct contents', () => {
+    const wrapper = render(ShareList);
+    expect(wrapper.find({ name: 'share-list' }).element).toBeDefined();
+  });
+});
```

Comment:
We're removing the lists feature (See #242). So you can delete these tests and not add any more tests for that feature :)

Reply:
Removed the tests for share-list. Could you please check now?

Reply:
Looks good to me. Let me know if you're planning on adding more tests to this PR

Reply:
Hey! I'm planning to add a few more tests to components and pages. I shall comment here once done

Reply:
@brenoferreira  Could you please check this?